packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/mvs.unit.test.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
+ *                                                                                 *
+ */
+
+/* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { FtpMvsApi } from "../../../src/ZoweExplorerFtpMvsApi";
+import { DataSetUtils } from "@zowe/zos-ftp-for-zowe-cli";
+import TestUtils from "../TestUtils";
+
+// two methods to mock modules: create a __mocks__ file for zowe-explorer-api.ts and direct mock for extension.ts
+jest.mock("../../../__mocks__/@zowe/zowe-explorer-api.ts");
+jest.mock("../../../src/extension.ts");
+
+const steam = require("stream");
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call
+const readbaleStream = steam.Readable.from([]);
```

Comment:
Consider using the constant names `stream` and `readableStream`. (The current spellings work well, but using the corrected spellings will make the code easier to maintain.)
- You might also want to make the same corrections in lines 25 and 27 of `packages/zowe-explorer-ftp-extension/__tests__/__unit__/Uss/uss.unit.test.ts`.

Reply:
Thank you @lauren-li ! I corrected the spelling mistakes. 😊